spec/controllers/registrations_controller_spec.rb

```diff
@@ -86,4 +86,45 @@
       expect(flash[:notice]).to eq I18n.t('devise.registrations.signed_up')
     end
   end
+
+  context 'default email:' do
```

Comment:
I feel like this check would be okay to be merged with the `signed_up_but_unconfirmed` test above.